tests/events_test.py

```diff
@@ -104,6 +106,23 @@ def test_commit_should_open_embryonic_event(self):
         events.commit(event)
         assert event.state == EventState.OPEN
 
+    def test_commit_should_remove_event_from_index_when_closing_event(self):
+        identifiers = "foobar", None, ReachabilityEvent
+        events = Events()
+        event = events.get_or_create_event(*identifiers)
+        events.commit(event)
+        event.set_state(EventState.CLOSED)
+        events.commit(event)
+        assert not events.get(*identifiers)
+
+    def test_commit_should_set_updated_when_closing_event(self):
+        identifiers = "foobar", None, ReachabilityEvent
+        events = Events()
+        event = events.get_or_create_event(*identifiers)
+        event.set_state(EventState.CLOSED)
+        events.commit(event)
+        assert (now() - timedelta(minutes=1)) < event.updated < (now())
```

Comment:
I'm left wondering: What value did `updated` have before the commit? Are they different? Should the test perhaps confirm that too?

Reply:
I am uncertain if by commit you mean the commit 77f2f5bbf37f6b1dcbbeac5215d141094e9aa9ec or you mean `events.commit(event)`.

If you mean the first we only so far set the `updated` when adding a log entry to an event.

If you mean the second then yes, `updated` would be `None` beforehand. 

But in either way, yes, if would be good to confirm that is actually changes.

Reply:
I meant `event.commit()`, yes.  And I think it might still be good to have it in the test, in case something changes `event.updated` before the commit, down the road.

Maybe the test should just set `event.updated` to a known value, and then after the commit assert that `event.updated` has indeed been changed to a newer timestamp.